common/utils/hostname.js

```diff
@@ -40,7 +39,7 @@ function getLocalMachineFullHostname() {
     else
     {
         // On Linux
-        var hostnameCmd = shell.exec('hostname -f');
+	var hostnameCmd = os.hostname(); 
         if (hostnameCmd.code !== 0)
```

Comment:
currently, we keep the full name in the DB and all API are using the full name, os.hostname  doesn't...